docs/populating-configuration.md

```diff
@@ -262,33 +262,13 @@ related to the Data Planes and its Kafka instances Fleet Manager needs to:
 * To send the credentials/information needed to setup the Observability Stack
   when it is installed in the Data Plane clusters
 
-In order for the Fleet Manager to be able to start, create the following files:
-```
-	touch secrets/rhsso-logs.clientId
-	touch secrets/rhsso-logs.clientSecret
-	touch secrets/rhsso-metrics.clientId
-	touch secrets/rhsso-metrics.clientSecret
-	touch secrets/observability-config-access.token
-```
-
 If you are not interested in making use of this functionality you can skip
 this section. Otherwise, keep reading below.
 
-The following command is used to setup the various secrets needed by
-the Observability stack:
-```
-OBSERVATORIUM_CONFIG_ACCESS_TOKEN="<observatorium-config-access-token> \
-RHSSO_LOGS_CLIENT_ID=<rhsso-logs-client-id> \
-RHSSO_LOGS_CLIENT_SECRET=<rhsso-logs-client-secret> \
-RHSSO_METRICS_CLIENT_ID=<rhsso-metrics-client-id> \
-RHSSO_METRICS_CLIENT_SECRET=<rhsso-metrics-client-secret> \
-make observatorium/setup
```

Comment:
Should we fully remove then the `observatorium/setup` Makefile target completely?

Reply:
I'll check if it does anything useful. If not we can remote it.

Reply:
I've removed it, it was only setting up those secrets.

Reply:
The tests are failing due to the removal. This is due to several Dockerfiles and/or CI scripts were referencing it. Could you review it?

thanks